react/src/components/atoms/forms/SelectBox/index.js

```diff
@@ -46,7 +46,7 @@ class SelectBox extends React.Component {
     const labelClassNames = stackLabel ? 'ma__select-box__label' : 'ma__label--inline ma__label--small';
     const selectBoxInline = stackLabel ? '' : 'ma__select-box__field--inline';
     const selectedIndex = this.props.options.findIndex((option) => option.text === selected);
```

Comment:
this still needs work, if the value selected from props is not in this.props.options, selectedIndex is undefined and component will break